README.md

```diff
@@ -1,3035 +1,87 @@
-
-  (vi) Add **CustomerEligibility** [*uml:v96-nk2; doc:v38.03*]
-   * ++   netex_customerEligibility_support-v1.1.xsd umlp
-   * ++    netex_customerEligibility_version-v1.1.xsd umlp
-* CR: CR00xx   Add **Presentation** including graphics to **AllowedLineDirection** [*uml:v96-nk2; doc:v38.03*]
-  * netex_line_version-v1.1.xsd umlp
-
-* CR: CR0040  Revise **Section**: Add **GeneralSection** distinct from **CommonSection**.   [*uml:v96-nk2; doc:v38.03*]
-     Separate out section  from point and link package. NB not strictly compatible just for **Section** usedIn **LinkSequence**
-  * netex_pointAndLinkSequence_support-v1.1.xsd  UMLcp
-  * netex_pointAndLinkSequence_version-v1.1.xsd  UMLcp
-  * netex_pointAndLink_support-v1.1.xsd UMLcp
-  * netex_pointAndLink_version-v1.1.xsd UMLcp
-  * netex_lineSection_version-v1.1.xsd UMLcp
-  * ++netex_section_support-v1.1.xsd UMLcp
-  * ++netex_section_version-v1.1.xsd UMLcp
-* CR: CR0010 **QuayType** Add _BusPlatform_ enum value [*uml:v96-nk2; doc:v38.03*]
-  * netex_ifopt_stopPlace_support-v1.1.xsd
-* CR: CR0030	Add **DayOffsets** [u*uml:v96-nk2; doc:v38.03*]
-  * netex_coupledJourney_version-v1.1.xsd UM p
-  * netex_datedPassingTime_version-v1.1.xsd
-  * netex_monitoredPassingTime_version-v1.1.xsd
-  * netex_passingTimes_version-v1.1.xsd
-* CR: CR0010 Add **JourneyPartPosition**  to **JourneyPart** [*uml:v96-nk2; doc:v38.03*]
-  * netex_coupledJourney_support-v1.1.xsd
-  * netex_coupledJourney_version-v1.1.xsd
-* CR: CR014 Add **GroupOfLinesType** enum [*uml:v96-nk2; doc:v38.03*]
-  * netex_line_support-v1.1.xsd
-  * netex_line_version-v1.1.xsd
-* CR: CR0047  Add support for tax to **FarePrice**: self ref on **PriceRule** & **StepResult** [*uml:v96-nk2; doc:v38.03*]
-  * netex_farePrice_version-v1.1.xsd
-  * netex_parkingTariff_support-v1.1.xsd
-  * netex_parkingTariff_version-v1.1.xsd
-
-===============================
-End
+## Releases
+| Release Number | Release Date  | Description                                    | Release Notes                                                                                   |
+| -------------- | ------------- | ---------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| 1.2            | March 2022    | Latest productive version before new modes merge | [Release Notes](https://github.com/NeTEx-CEN/NeTEx/releases/tag/v1.2)      |
+| 1.2.2          | August 2023   | First version of new modes                     | [Release Notes](https://github.com/NeTEx-CEN/NeTEx/releases/tag/v1.2.2)        |
+| 1.2.3          |               | Bug fixes (when necessary)                     | [Release Notes](https://github.com/NeTEx-CEN/NeTEx/blob/next/CHANGELOG.md)       |
+| 1.3.0          | Dec. 2023     | Upcoming release   ⏳                            | N/A                                                                                             |
```

Comment:
```suggestion
| 2.0          | 2024     | Upcoming release   ⏳                            | N/A  | N/A                                                                                             |
```

Reply:
I hope this works and yields a correct table.

Reply:
Not clear to me when considering the change_log.md below: will the above referenced Release Notes become obsolete?